projects/ngneat/dialog/src/lib/specs/dialog.component.spec.ts

```diff
@@ -190,6 +190,12 @@ describe('DialogComponent', () => {
       dialogService.dialogs.push(Object.assign(spectator.component.dialogRef, { id: 'close-last-open-dialog' }));
     });
 
+    it('should properly update dialog config', () => {
```

Comment:
can we please have more check? for example, let's update the width and change enableClose to false and see if it works

Reply:
I have added more testes let's me know whether they cover what expected